# >0 : Clockwise rotation
# <0 : Anticlockwise rotation
float32 gimbal_rotate

# true : when gimbal == 0, grab the target
# false : stand by
bool arm_moveit